packages/contractkit/src/providers/celo-provider.test.ts

```diff
@@ -1,6 +1,6 @@
 import { privateKeyToAddress } from '@celo/utils/lib/address'
-import { Tx } from 'web3/eth/types'
-import { Callback, JsonRPCRequest, JsonRPCResponse, Provider } from 'web3/providers'
+import { provider, Tx } from 'web3-core'
```

Comment:
nit: capitalized type name

Reply:
This `provider` is a type that they declared in as one of `their providers`, `string` or `null`.
They erase the `Provider` interface.

They made a lot of damage to that interface... I tried to "get this working", but probably we should think if web3 with its unnecessary not unified code code and its radicals changes, is the best lib for us